### ISSUE TITLE
Fear Effect not loading (PS3)

### ISSUE THREAD

**Issue by nydalap** [sahlberg/pop-fe#39]
Hello
 I have an issue with Fear Effect NTSC-U SLUS00920, SLUS01056, SLUS01057, SLUS01058. When I convert the bin/cue files and try to play the game, it freezes at the PlayStation logo. When I convert Fear Effect PAL  SLES02166, SLES12166, SLES22166, SLES32166 I get a black screen. I checked the CRC-32 and the MD5, and they match the PAL and the NTSC versions of the discs. Is there something I'm doing wrong, or am I missing something.    
 Best regards
 
 Update
I deleted the "pop-fe-ui" folder and downloaded it again. The PAL version is working, but the NTSC version still freezes at the PlayStation logo.

**Comment by nydalap** [comment]
Update
 I deleted the "pop-fe-ui" folder and downloaded it again. The PAL version is working, but the NTSC version still freezes at the PlayStation logo.

**Comment by sahlberg** [comment]
I will have a look tomorrow and test the NTSC version.

**Comment by sahlberg** [comment]
I can reproduce the hang with a PKG made on linux.
It does hang at the PS1 logo when trying to boot the game (from disc 1).
It doesn't look like there is an issue with pop-fe itself but rather that something on disc-1 itself boot process is just not compatible with the emulator ps1_netemu.

I found a workaround though,
when it hangs at the PS1 logo, use the PS button and then reset the game with disc 2.
It will then re-start using the second disk. At the menu, when you select "new game" it will
prompt you for disc number 1. Press the PS button again and switch disc to disc 1 and press X back on the menu.
It will load disc 1 and start the cutscene at the beginning of the game.

A little convoluted but at least it works.   The emulator ps1_netemu is not perfect and not 100% compatible :-(  but at least in this situation there is a workaround.

**Comment by sahlberg** [comment]
It seems like disc1 of Fear Effect NTSC-U does something that breaks the emulator so it is unlikely we can do much.

There are two workarounds though you can use
1, When it hangs, reset the game to boot disc2, then at the start menu it will prompt you to insert disc1 and load it.
This will bypass the EDIOS/FMV that plays on boot of disc1 and which may be causing the issue.
2, You can create a PKG using the European version of Fear Effect and use the "Force NTSC" mode.
This works and creates a game that plays in NTSC settings, i.e. resolution and speed, but avoids the issue that the NTSC-U version of the game triggers.

As I am unlikely to be able to do anything in pop-fe to avoid this, and since I don't have the skills to reverse engineer or patch sony's emulator I will have to close this issue.
But at least, you have two alternative workarounds to get the game to load and play eventhough it is a little klunky.

**Comment by nydalap** [comment]
How can I "Force NTSC" mode?

**Comment by sahlberg** [comment]
Use the pop-fe-ui artefact from this link:
https://github.com/sahlberg/pop-fe/actions/runs/6081132085

the pop-fe-ps3 UI has a checkbox for it

**Comment by nydalap** [comment]
Thanks  for the workarounds that helped in  solving the issues.
 Best regards

**Comment by sahlberg** [comment]
Actually, I asked someone that knows a lot more about EBOOT.PBP than me and is a legendary hacker for PS1 stuff and he told me that it was in fact a bug in pop-fe and I generated an incorrect TOC that caused this game to fail.

I have fixed that now and now the NTSC-U version of Fear Effect works correctly for disc 1 and there is no hang.
You can download the latest version and your game should work now as it should now without any need for workarounds.

**Comment by nydalap** [comment]
Hello 
I have downloaded the newest version https://github.com/sahlberg/pop-fe/tree/v1.3 and created a PKG. Disc 1 is now booting up, but when  I press start at the title screen, it fades in and the main menu does not appear. The screen is black, and the music is still playing.  I created a new PKG with the option "Force PS1 new emu" and this one works. It  does not hang when I press start  and the main menu is displaying correctly. I finished the first disc without any bugs or issues.
Best regards

**Comment by sahlberg** [comment]
That is good that it works with ps1_newemu.
Can you report this on this thread : https://www.psx-place.com/threads/ps1_netemu-external-configs-and-ad-hoc-params.41504/page-4#post-372204
and maybe we can find if there is a config setting that makes it work also on the default emulator.

**Comment by nydalap** [comment]
I am unable to post there, it says that I  do not have sufficient privileges to reply in that post.

**Comment by sahlberg** [comment]
Weird. Maybe PM one of the mods?
I will post our results there though because mrjaredbeta and the others that work on the configs post there and they might be able to find a config that solves the issue.

**Comment by sahlberg** [comment]
Can you test the build https://github.com/sahlberg/pop-fe/actions/runs/6242658676
which contains the config that mrjaredbeta found.
It should now work without the need to use ps1_newemu.

Thanks for this report. We fixed yet another game to work on the ps3!

**Comment by nydalap** [comment]
Hello 
 I created a PKG using pop-fe from the link You pasted. The first time I ran the game, the title screen works when I pressed start. When I selected options, the screen turned black and there was only music playing. After resetting the game the options screen loaded I turned on the subtitles and increased the master volume but when I wanted to return to the main menu the game froze again and I got black screen and the music playing. I reset the game again and now options work I can turn on the subtitles and increased the master volume and return to the main menu without the game freezing.  I played the game for 15 minutes, saved it and exited to test the option screen, and it sometimes works and sometimes still freezes the game and the music is not playing.
